### ISSUE TITLE
Best practices for today

### ISSUE THREAD

**Issue by AdamBrodzinski** [rschultheis/rspec_capybara_starter#1]
First thanks for putting this together, it's great! I've used it in the past and it's worked rather well.

I'm starting a new project and I was wondering if there's anything I should update or tweak based on what's the best solution for today?

It doesn't seem like Rspec or Capy have changed much since the gemfile version but I was going to try and bump those as well. If this works would you want a PR for the gemfile bump?

Also is there an easy way to parallelize the tests for each core? This isn't a deal break but thought it might be worth adding.

**Comment by rschultheis** [comment]
@AdamBrodzinski are you still using this project?  My apologies for not responding sooner to this nearly 2 year old issue!  I've updated the project recently as it has gotten some renewed interest from a few sources.  Biggest difference is I dropped PhantomJS and added headless chrome.

I'll respond to issue going forward, in the past I simply wasn't setup to get notifications of new issues.

Best,
Rob
